pkgs/development/python-modules/black/default.nix

```diff
@@ -23,14 +23,19 @@ buildPythonPackage rec {
 
   # Don't know why these tests fails
   # Disable test_expression_diff, because it fails on darwin
+  # Disable Primer* tests because they depend on black beeing in PATH
```

Comment:
Wouldn't it be better just to add black to the path for the purposes of this test?

Reply:
Sorry for the late reply. Since I'm fairly new to the nix world I'm not sure how to do this. Something like 
```
  preCheck = ''
    export PATH=${black}
  '';
```
will obviously not work

Reply:
I saw some examples of
```
  preCheck = ''
    export PATH="$PATH:$out/bin"
  '';
```
in nixpkgs.